### ISSUE TITLE
Showing help inside demo crashes program

### ISSUE THREAD

**Issue by gradha** [domesticcatsoftware/DCIntrospect#17]
Just grabbed a copy of DCIntrospect. I tried compiling the demo example and run it on Xcode 4.0.2 with the iPhone 4.3 simulator. The example shows and I press the space bar to enter the introspection mode. Everything seems to work fine, but pressing ? to show the help screen crashes the application. The crash sometimes happens immediately, other times it seems to take a few seconds to trigger, but the backtrace is always the same:

```
#0  0x00000000 in ?? ()
#1  0x00003d71 in UITextInputTraits_valueForKey (self=0x8866800, _cmd=0x38c62b2, key=0x4f6f760) at DCIntrospect.m:87
#2  0x04b676bf in -[NSObject(UIAccessibilitySafeCategory) safeValueForKey:] ()
#3  0x04f22377 in -[UIWebDocumentViewAccessibility(SafeCategory) webView:didFinishLoadForFrame:] ()
#4  0x00ebbc7d in __invoking___ ()
#5  0x00ebbb51 in -[NSInvocation invoke] ()
#6  0x00ee9858 in -[NSInvocation invokeWithTarget:] ()
#7  0x00ebca04 in ___forwarding___ ()
#8  0x00ebc522 in __forwarding_prep_0___ ()
#9  0x00ebbc7d in __invoking___ ()
#10 0x00ebbb51 in -[NSInvocation invoke] ()
#11 0x036cf140 in HandleDelegateSource ()
#12 0x00f2c8ff in __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ ()
#13 0x00e8a88b in __CFRunLoopDoSources0 ()
#14 0x00e89d86 in __CFRunLoopRun ()
#15 0x00e89840 in CFRunLoopRunSpecific ()
#16 0x00e89761 in CFRunLoopRunInMode ()
#17 0x01a921c4 in GSEventRunModal ()
#18 0x01a92289 in GSEventRun ()
#19 0x001abc93 in UIApplicationMain ()
#20 0x00002bc9 in main (argc=1, argv=0xbfffe758) at main.m:14
```

**Comment by 0xced** [comment]
This crash will be fixed once #19 is pulled.

**Comment by gradha** [comment]
Thanks!
